### PR TITLE
feat(fe): add people article detail page

### PR DESCRIPTION
**Pull request by willwei954**

### CODE REVIEW

File: fe/src/pages/peopleDetail.page.tsx
Comment (SeanWong24) [inline fe/src/pages/peopleDetail.page.tsx:1]: rename it to `people-detail.page.tsx` to keep consistent with `language-toggle.view.tsx` or rename all components like `PeopleDetail.page.tsx`.

File: fe/src/pages/peopleDetail.page.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        />
+      </div>
+    </>
+  );
+}
+
+function ImageDisplayer(props: { imageQueue: string[]; height?: number }) {
+  const { imageQueue, height } = props;
+
+  const [selectedIndex, setSelectedIndex] = useState(0);
```
Comment (SeanWong24) [inline fe/src/pages/peopleDetail.page.tsx]: use signal instead

File: fe/src/views/language-toggle.view.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ export function LanguageToggleView() {
     <Tabs
       className="w-fit"
       value={`${activeLanguage.value}`}
-      onTabChange={(e) => {
+      onTabChange={(e: any) => {
```
Comment (SeanWong24) [inline fe/src/views/language-toggle.view.tsx]: use `e: CustomEvent<unknown>` instead

File: fe/src/pages/peopleDetail.page.tsx
Comment (SeanWong24) [inline fe/src/pages/peopleDetail.page.tsx:1]: Did not find an entrance to this page from the UI.

File: fe/src/pages/peopleDetail.page.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        <Markdown
+          markdown={markdown.value}
+          styleSheets={[markdownStyleSheet]}
+        />
+      </div>
+    </>
+  );
+}
+
+function ImageDisplayer(props: { imageQueue: string[]; height?: number }) {
```
Comment (SeanWong24) [inline fe/src/pages/peopleDetail.page.tsx]: It seems that the `height` has never been used.

File: fe/src/pages/peopleDetail.page.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const maxHeight = height ?? "30vw";
+
+  return (
+    <div class={`grid grid-cols-5 gap-5`} style={{ height: maxHeight }}>
+      <div class="m-auto  col-span-4">
+        <img
+          class={`max-w-100% max-h-${maxHeight}`}
```
Comment (SeanWong24) [inline fe/src/pages/peopleDetail.page.tsx]: The max height is set in both class and style. If the `max-h-${variable}` does not work, just remove it. But can also consider using CSS function like `min(100%, 30vw)`.

File: fe/src/pages/peopleDetail.page.tsx (hunk 3 above)
Comment (SeanWong24) [inline fe/src/pages/peopleDetail.page.tsx]: Maybe this can be extracted into a separate view.

File: fe/src/pages/peopleDetail.page.tsx
Comment (willwei954) [reply fe/src/pages/peopleDetail.page.tsx:1]: this will be done in a different PR.

**Comment by SeanWong24** [comment]
#29

**Review by SeanWong24 — APPROVED** [review]

**Pull request merged**